webpack.config.js

```diff
@@ -61,7 +61,7 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
   },
   optimization: {
     chunkIds: 'natural',
-    splitChunks: { chunks: 'all' },
+    splitChunks: { chunks: (chunk) => chunk.name !== 'polyfill' },
```

Comment:
The reason for this is to ensure that `polyfill` is kept as a single, self-contained pack, since we don't (and cannot reasonably) share default script chunk rendering between the polyfill and other scripts, due to the need for `nomodule` attribute on the polyfill script.